lib/index.d.ts

```diff
@@ -37,23 +37,6 @@ export type ServiceClientOptions = {
     timeout?: number;
 };
 
-export interface ServiceRequest {
-    headers: Headers;
-    logger: Logger;
```

Comment:
This would require use of pino, which is not a requirement to use service-client